bundles/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/config/EnoceanGenericBindingProvider.java

```diff
@@ -77,17 +77,6 @@ protected void addBindingConfig(Item item, BindingConfig config) {
     }
 
     @Override
-    public void removeConfigurations(String context) {
-        Set<Item> configuredItems = contextMap.get(context);
-        if (configuredItems != null) {
-            for (Item item : configuredItems) {
-                items.remove(item.getName());
-            }
-        }
-        super.removeConfigurations(context);
-    }
-
-    @Override
     public EnoceanParameterAddress getParameterAddress(String itemName) {
```

Comment:
This override performed an unnecessary one-by-one removal of items from the contextMap, but introduced problem where listeners would not be called and items would not be removed from bindingConfigs map.  Removing method altogether should remove the bug.